docs/dev/docker-hub.md

```diff
@@ -314,6 +314,14 @@ For example, to disable developer mode:
 $ docker run --name some-scylla -d scylladb/scylla --developer-mode 0
 ```
 
+#### `--default-log-level arg (=info)``
```

Comment:
we also have https://github.com/scylladb/scylladb/blob/master/docs/troubleshooting/log-level.rst . i think ideally, this document should contains the content more relevant to the usage of scylla used along with container image instead of replicating the documents, what do you think? if we really want to enrich this document with related information, probably we can add a link?